en-US/snippets/2017/sep2017_a.lang

```diff
@@ -0,0 +1,122 @@
+## NOTE: Reference bug 1392702. Deadline: 4 Sept, 2017.
+## NOTE: Guidelines on localizing snippets: https://goo.gl/nEhdt4. The tone is QUAC (Quick, Untamed, Alert, Clever). Do your best to make it local and culturally relevant.
+## NOTE: No character length limit, but try not to exceed English length.
+
+
+# Snippet for Firefox Screenshots
+# Link: https://screenshots.firefox.com/#hello
+# "Complex screenshot hack" refers to traditional and more tedious method of taking and sharing screenshots.
+;Say goodbye to you complex screenshot hack. Screenshots is a new way to save and share within Firefox. <a>Learn more.</a>
+Say goodbye to you complex screenshot hack. Screenshots is a new way to save and share within Firefox. <a>Learn more.</a>
+
+
+# Snippet for Firefox Screenshots
+# Link: https://screenshots.firefox.com/#hello
+# "Cool" in this context means the quality of being fashionably attractive or impressive. This message intends to convey that Firefox Screenshots is an easy way to save and share interesting images online.
+;Screenshots is a better way to screenshot. Save and share cool stuff without leaving Firefox. <a>Learn More.</a>
+Screenshots is a better way to screenshot. Save and share cool stuff without leaving Firefox. <a>Learn More.</a>
+
+
+# Snippet for Firefox 55
+# Link:https://support.mozilla.org/kb/performance-settings
+# This message highlights that users are in control and they have the ability to customize Firefox.
+;1 processor, 4 processors, 7 processors? In Firefox 55, <a>the choice is yours.</a>
+1 processor, 4 processors, 7 processors? In Firefox 55, <a>the choice is yours.</a>
```

Comment:
(unrelated to this PR) I'm not sure if this wanted to be funny (process, processor), but it turns out to be misleading: you can set the number of content processes, number of processors is something else :-\